db/migrate/20160807214215_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :provider, :string, :null => false, :default => "email"
+    add_column :users, :uid, :string, :null => false, :default => ""
+
+    ## Confirmable
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
```

Comment:
Needed this for the `devise_token_auth` gem we're using. There's some hacky things I'm doing to get it to work. We may go with a different solution later on, but for now, this is fine.